server/rest/src/test/java/org/infinispan/rest/resources/CacheResourceV2Test.java

```diff
@@ -695,8 +695,15 @@ public void testCacheFullDetail() {
       assertThat(document.at("indexing_in_progress")).isNotNull();
       assertThat(document.at("queryable")).isNotNull();
       assertThat(document.at("rebalancing_enabled")).isNotNull();
-      assertThat(document.at("key_storage").asString()).isEqualTo("application/unknown");
-      assertThat(document.at("value_storage").asString()).isEqualTo("application/unknown");
+      assertThat(document.at("key_storage").asString()).isEqualTo("application/x-protostream");
+      assertThat(document.at("value_storage").asString()).isEqualTo("application/x-protostream");
+      assertThat(document.at("key_encoding").asString()).isEqualTo("application/x-protostream");
+      assertThat(document.at("value_encoding").asString()).isEqualTo("application/x-protostream");
+      assertThat(document.at("mode").asString()).isEqualTo("DIST_SYNC");
+      assertThat(document.at("storage_type").asString()).isEqualTo("HEAP");
+      assertThat(document.at("max_size").asString()).isEmpty();
+      assertThat(document.at("max_size_bytes").asLong()).isEqualTo(-1);
+
```

Comment:
Should we also adds assertion for when `security == true` to ensure that `storageType`, `maxSize` and `maxSizeBytes` are not set?

Reply:
done @ryanemerson